### ISSUE TITLE
Cannot find module './build/Debug/DTraceProviderBindings'

### ISSUE THREAD

**Issue by calclavia** [auth0/wt-cli#78]
When running any wt-cli commands, I get the following error.

```
{ Error: Cannot find module './build/Debug/DTraceProviderBindings'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/wt-cli/node_modules/dtrace-provider/dtrace-provider.js:17:23)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/wt-cli/node_modules/bunyan/lib/bunyan.js:79:18)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/wt-cli/lib/logs.js:1:76)
    at Module._compile (module.js:541:32) code: 'MODULE_NOT_FOUND' }
```

**Comment by jschluchter** [comment]
@calclavia I get the same thing, however the commands do seem to finish successfully.

**Comment by jasonbarone** [comment]
I had a very similar error after a version update. Make sure to try uninstalling the package completely, then reinstalling. That fixed mine.

**Comment by jschluchter** [comment]
@jasonbarone that resolved the errors. However, any new user might encounter this scenario.

**Comment by kevinbluer** [comment]
Uninstall / reinstall worked for me too 👍

**Comment by darkyen** [comment]
This might also occur if you recently changed the version of Node.JS

**Comment by glrodasz** [comment]
It is happening with **Node v6.95** and global packages:

build@0.1.4
essential@0.0.1
npm@3.10.10
wt-cli@4.0.4

**Comment by ggoodman** [comment]
This will be fixed by #111.

**Comment by ggoodman** [comment]
Closed by #111
